src/igvfd/types/file.py

```diff
@@ -1147,9 +1147,9 @@ def validate_bucket_location(request, properties, bucket):
 )
 def post_upload(context, request):
     properties = context.upgrade_properties()
-    if properties['upload_status'] == 'validated':
+    if properties['upload_status'] not in ['pending', 'file not found', 'invalidated']:
         raise HTTPForbidden(
-            'Unable to issue new credentials when uploading_status is validated'
+            'Unable to issue new credentials when uploading_status is validated or validation exempted'
```

Comment:
Now that we are touching this, could you please fix the old typo in the error message and change `uploading_status` to the correct `upload_status`, please?